docs/addons/calico.md

```diff
@@ -0,0 +1,190 @@
+# Calico Add-on
+
+The `Calico` addon adds support for [Calico CNI](https://docs.projectcalico.org/about/about-calico) to an EKS cluster.
+
+Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. To secure workloads in Kubernetes, Calico utilizes Network Policies as we will see below.
+
+## Usage
+
+```typescript
+import { addons }  from '@shapirov/cdk-eks-blueprint';
+
+const myCalicoCNI = new addon.CalicoAddon();
+const addOns: Array<ClusterAddOn> = [ myCalicoCNI ];
+
+const app = new cdk.App();
+new EksBlueprint(app, 'my-stack-name', addOns, [], {
+  env: {
+      account: <AWS_ACCOUNT_ID>,
+      region: <AWS_REGION>,
+  },
+});
+```
+## Securing your environment with Calico network policies
+
+### Installing calicoctl
+
+In order to use Calico Customer Resource Definitions (CRD), you need to install `calicoctl`. 
```

Comment:
It is Custom (not customer)

Reply:
Let's add a sentence on why customer would want to have custom resource definitions. We should state that by default just installing Calico for network policy support will enable customers to define standard k8s policies. However, Calico also allows custom extensions (examples). For that, customer may consider installing the CLI.

Reply:
blurb added.